### ISSUE TITLE
somatic indel of high AF can't be called by strelka2, but vardictJava can, why?

### ISSUE THREAD

**Issue by ShannonDaddy** [Illumina/strelka#40]
Hi, 
I have a high-depth amplicon sequencing data of a tumor-normal paired samples. I use strelka2 to call somatic mutations of LC1801 whose negative control is LC18NC, and I found that strelka2 called far less mutations than vardictJava. Particularly, the variant "chr7    55242464        rs121913421;163343;COSM6223     AGGAATTAAGAGAAGC        A"  with a AF about 10% is missing by strelka2, while called by vardictJava. I'm very confused about the results, I wonder are there any parameters of streka2 which can be tuned to be more sensitive. I can send the testing data and running scripts to you to reproduce the issue if you could tell me your email, my email is gqian@lc-bio.com. I have also uploaded the data to the baidu netdisk which is very popular in china, the download link is https://pan.baidu.com/s/1AdZVKoJjXQO3ldpkUcNn2g  with password: yher
Can you check that for me please ? Thanks a lot.

**Comment by ctsa** [comment]
Thanks for reporting, looking into this case now.

**Comment by ctsa** [comment]
I've taken a quick look at this data: The immediate issue is that all of the read pairs in this region start at the same position (eg, for most read fragments supporting the above 15 base deletion, R1 and R2 both start at 55,242,433) Such short fragments are not marked as proper pairs by the mapper, thus strelka will skip these for the purpose of nominating a somatic variant. Such pairs will only be used at the 'tier2' evidence level, but this is only used to disprove a somatic hypothesis by finding additional evidence in the normal sample suggesting the variant could be germline.

While there are some simple steps we could take to address this, the best way to proceed may be to look into a method with a greater targeted/amplicon focus: for instance (as you have already examined) the Pisces somatic caller here: https://github.com/Illumina/Pisces

**Comment by ShannonDaddy** [comment]
Thanks, chris. 

I checked these sam records again, but I found that the sam flags of them are 99 or 147, which mean they are proper paired.  Does streka check 'proper paired' from the sam flags or somewhere else? 

cons:168234     99      chr7    55242432        60      100M    =       55242432        100     GGGTGAGAAAGTTAAAATTCCCGTCGC
cons:168234     147     chr7    55242432        60      100M    =       55242432        -100    GGGTGAGAAAGTTAAAATTCCCGTCGC
cons:168235     99      chr7    55242432        60     100M    =       55242432        100     TGGTGNGAAAGTTAAAATTCCCGTCGC
cons:168235     147     chr7    55242432        60      100M    =       55242432        -100    TGGTGNGAAAGTTAAAATTCCCGTCGC
cons:168236     99      chr7    55242432        60      33M15D52M15S    =       55242433        100     TGGTGNGAANGTTAAAATT
cons:168236     147     chr7    55242433        60      16S32M15D52M    =       55242432        -100    CTATCAATCCAATACTGGT
cons:168237     99      chr7    55242432        60      33M15D52M15S    =       55242433        100     TGGNGAGAAAGTTAAAATT
cons:168237     147     chr7    55242433        60      16S32M15D52M    =       55242432        -100    CTAACAACCAACCCCTGGT
